public/components/health-check/health-check.tsx

```diff
@@ -118,22 +118,29 @@ export class HealthCheck extends Component {
       const response = await GenericRequest.request('GET', '/hosts/apis');
       const hosts = response.data;
       const errors = [];
+      const results = this.state.results;
 
       if (hosts.length) {
         for (let i = 0; i < hosts.length; i++) {
-          let tries = 36;
+          const maxTries = 5;
```

Comment:
`maxTries` is declared for every loop on for?

Reply:
yes, your are right